packages/react-aria-components/src/Menu.tsx

```diff
@@ -343,7 +343,6 @@ export const MenuItem = /*#__PURE__*/ createLeafComponent('item', function MenuI
     selectionManager
   }, state, ref);
 
-  let {isFocusVisible, focusProps} = useFocusRing();
```

Comment:
removing this means we won't respond to the isFocusVisible state change anymore, it'll be only driven by and checked during renders, so we may miss an interaction mode change right?

Reply:
yeah, I suppose so but `useFocusRing` only updated on focus changes anyways, which meant that the focus ring would remain when you hovered the currently focused item despite the modality being pointer at that point. Now the focus ring will disappear when hovering a previously keyboard focused item. Open to discussion as to whether or not we consider that a bug fix or a unintended change in functionality. One thing to note is that Listbox behaves the same way

Reply:
Listbox behaves as Menu doesn in this PR you mean?

Reply:
yep, thats correct. The change here brings Menu in line with Listbox behavior